nservicebus/pipeline/unit-of-work.md

```diff
@@ -6,6 +6,19 @@ redirects:
 - nservicebus/unit-of-work-in-nservicebus
 ---
 
+## Avoiding partial updates
```

Comment:
Perhaps the title should be more like "Using a transaction scope" and the partial updates can be moved into the text instead as a use case?